apps/coscrad-frontend/src/components/resources/vocabulary-lists/vocabulary-list-form.tsx

```diff
@@ -14,11 +14,13 @@ export const VocabularyListForm = ({
     onFormChange,
     formState,
 }: VocabularyListFormProps): JSX.Element => (
-    <Card>
-        <CardHeader title="Filter the Vocabulary List" />
-        <CardContent>
+    <Typography component={'div'}>
+        <Typography variant="h4" component={'div'}>
+            Filter the Vocabulary List
+        </Typography>
+        <Typography component={'div'} sx={{ margin: 'auto', textAlign: 'center' }}>
```

Comment:
Again, I think this should be a `Box` or `Grid` for layout.

Reply:
Resolved.